Source/Scene/Cesium3DTileset.js

```diff
@@ -1016,6 +1016,10 @@ function Cesium3DTileset(options) {
     .then(function (tilesetJson) {
       // This needs to be called before loadTileset() so tile metadata
       // can be initialized synchronously in the Cesium3DTile constructor
+      const metadataJson = tilesetJson.metadata;
+      if (!defined(metadataJson)) {
+        return processMetadataExtensionLegacy(that, tilesetJson);
+      }
       return processMetadataExtension(that, tilesetJson);
```

Comment:
This is super nitpicky, but I would invert the logic of this if statement:

```
if (defined(metadataJson)) {
  // process metadata extension
} else {
  // process legacy extension
}
```

Just for consistency with metadata parsing elsewhere (parse new stuff formats, fall back to legacy) and readability (no negative)